### ISSUE TITLE
Dataset from slices, split to train, test

### ISSUE THREAD

**Issue by kramorov** [kramorov/kramorov#4]
# Create the tf.data.Dataset from the existing data
dataset = tf.data.Dataset.from_tensor_slices((x_train, y_train))

# Split the data into a train and a test set.
SAMPLES_TRAINING = int(len(x_train)*0.8)
train_dataset = dataset.take(SAMPLES_TRAINING)
test_dataset = dataset.skip(SAMPLES_TRAINING)

# Both datasets have to be repeated and batched appropriately
train_dataset = train_dataset.repeat().batch(BATCH_SIZE)
test_dataset = test_dataset.repeat().batch(BATCH_SIZE)
